scripts/src/javatests/org/oppia/android/scripts/maven/GenerateMavenDependenciesListTest.kt

```diff
@@ -550,6 +555,394 @@ class GenerateMavenDependenciesListTest {
     )
   }
 
+  @Test
+  fun testLicensesIncomplete_noUpdatesProvided_scriptFails() {
+    val textProtoFile = tempFolder.newFile("scripts/assets/maven_dependencies.textproto")
+    val pbFile = tempFolder.newFile("scripts/assets/maven_dependencies.pb")
+
+    val incompleteApacheLicense = License.newBuilder().apply {
+      this.licenseName = "The Apache License, Version 2.0"
+      this.originalLink = "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }.build()
+    val incompleteApacheLicenseWithSameOriginalLink = License.newBuilder().apply {
+      this.licenseName = "The Apache Software License, Version 2.0"
+      this.originalLink = "https://www.apache.org/licenses/LICENSE-2.0.txt"
+    }.build()
+    val completeBsdLicense = License.newBuilder().apply {
+      this.licenseName = "Simplified BSD License"
+      this.extractedCopyLink = ExtractedCopyLink.newBuilder()
+        .setUrl("https://local-copy/bsd-license").build()
```

Comment:
```suggestion
      this.extractedCopyLink = ExtractedCopyLink.newBuilder().apply {
        url = "https://local-copy/bsd-license"
      }.build()
```

The formatting of the previous version is a bit off, but this alternative also makes use of apply.

Ditto for other similar cases below.

Reply:
Done.